r2/r2/templates/linkinfobar.html

```diff
@@ -39,7 +39,7 @@
       &#32;
     </span>
     <time datetime="${html_datetime(thing.a._date)}">
-      ${thing.a._date.strftime(thing.datefmt)}
+      ${thing.a._date.strftime(_('%d %b %Y'))}
```

Comment:
Do you mean something like

    format_date(thing.a._date, format=_('dd MMM yyyy'), locale=_('en_US'))

How do I get the locale name?